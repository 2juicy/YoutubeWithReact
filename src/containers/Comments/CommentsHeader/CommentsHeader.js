import React from "react";
import "./CommentsHeader.scss";
import { Button, Icon } from "semantic-ui-react";

export default function Comments({ amount }) {
  return (
    <div className="comment-header">
      <h4>{amount} Comments</h4>
      <Button basic compact icon labelPosition="left">
        <Icon name="align left" />
        Sort by
      </Button>
    </div>
  );
}
