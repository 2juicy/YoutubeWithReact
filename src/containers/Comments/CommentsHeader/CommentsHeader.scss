.comment-header {
  h4 {
    display: inline-block;
    margin-right: 16px;
  }
}
