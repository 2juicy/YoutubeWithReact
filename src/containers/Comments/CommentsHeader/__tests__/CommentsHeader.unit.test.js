import React from "react";
import { shallow } from "enzyme";
import CommentsHeader from "../CommentsHeader";

describe("CommentsHeader", () => {
  test("renders without props", () => {
    const wrapper = shallow(<CommentsHeader />);
    expect(wrapper).toMatchSnapshot();
  });

  test("renders with amount", () => {
    const wrapper = shallow(<CommentsHeader amount={12345} />);
    expect(wrapper).toMatchSnapshot();
  });
});
