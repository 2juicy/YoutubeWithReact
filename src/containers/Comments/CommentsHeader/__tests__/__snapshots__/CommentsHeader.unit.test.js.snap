// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentsHeader renders with amount 1`] = `
<div
  className="comment-header"
>
  <h4>
    12345
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;

exports[`CommentsHeader renders without props 1`] = `
<div
  className="comment-header"
>
  <h4>
     Comments
  </h4>
  <Button
    as="button"
    basic={true}
    compact={true}
    icon={true}
    labelPosition="left"
  >
    <Icon
      as="i"
      name="align left"
    />
    Sort by
  </Button>
</div>
`;
